### PR TITLE
fix: adjust layout and spacing in Home/SlidableList

### PR DESCRIPTION
**Pull request by luis-herasme**
### Checklist

- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [ ] I have filled out the description and linked the related issues.

### Description
Adjust layout and spacing in Home/SlidableList

### Screenshots (if appropriate):
Before:
image
image

After:
image
image

### CODE REVIEW

**Review by PJColombo — APPROVED** [review]
lgtm

File: apps/web/src/pages/index.tsx [hunk 1]
```diff
@@ -191,7 +191,7 @@ const Home: NextPage = () => {
         <div className="grid grid-cols-1 items-stretch justify-stretch gap-6 lg:grid-cols-3">
           <Card
             header={
-              <div className=" flex flex-wrap items-center justify-between gap-5">
+              <div className="flex-warp flex flex-col justify-between gap-3 2xl:flex-row 2xl:items-center">
```
Comment (xFJA) [inline apps/web/src/pages/index.tsx:194]:
```suggestion
              <div className="flex-wrap flex flex-col justify-between gap-3 2xl:flex-row 2xl:items-center">
```

**Review by xFJA — COMMENTED** [review]
Left a comment about a TW class typo

File: apps/web/src/pages/index.tsx (hunk 1 above)
Comment (luis-herasme) [reply apps/web/src/pages/index.tsx]: Done [dfa879b](https://github.com/Blobscan/blobscan/pull/632/commits/dfa879b98d84863a32a63d819bb887622bf812f5)

**Pull request merged**